README.md

```diff
@@ -146,3 +146,6 @@ To access PhpMyAdmin, run `docker ps` to get its dynamic port.
 * Username: root
 * Password: root
 
+### SOLR Integration
+To enable SOLR, uncomment corresponding lines from docker-compose.yml file. Current version is 7.6.0, to use a different version change the SOLR_URL value with the preferred release version in the SOLR Dockerfile. Releases are available in https://archive.apache.org/dist/lucene/solr/. SOLR can be accessed in [local-domain]:8983/solr/#/ ( e.g. http://192.168.99.100:8983/solr/#/ )
```

Comment:
Should users rather copy and paste the Dockerfile from the Github repo where SOLR is maintained? https://github.com/docker-solr/docker-solr
Looks like SOLR_VERSION ... etc would also need to be updated, not only SOLR_URL.